openapi.yaml

```
@@ -360,6 +360,34 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /statistics/snapshot:
+    post:
+      summary: Increments download statistics
+      description: Increments snapshot download statistics
```

Maybe this is a more accurate formulation
```suggestion
      summary: Records snapshot download event
      description: Records snapshot download event
```